catkin_ws/src/sensors/launch/sensors_status.launch

```diff
@@ -1,5 +1,10 @@
 <launch>
      <param name="sensor_warning_interval" value="5"/> <!-- secs -->
      <param name="sensor_status_update_rate" value="10"/> <!-- Hz -->
+     <!-- Frequency of each pinger -->
+	<param name="pinger_frequency_1" value="1"/>
```

Comment:
these will be like 20_000, 25_000 in the future right?

Reply:
lets just put them to what they will be at comp right now tbh

Reply:
do you know what they are? I couldn't find the exact values in the competition document. They only give a range. I was hoping electrical would know

Reply:
ah ok i thought we had the values, but yeah I checked last year's and they said the values changed every day and were announced at competition